lib/Serialization/Deserialization.cpp

```diff
@@ -1343,9 +1343,13 @@ ModuleFile::resolveCrossReference(ModuleDecl *baseModule, uint32_t pathLen) {
             nestedType = containingFile->lookupNestedType(memberName, baseType);
           }
         } else {
-          // Fault in extensions, then ask every serialized AST in the module.
+          // Fault in extensions, then ask every file in the module.
+          ModuleDecl *extensionModule = M;
```

Comment:
`M` here is the module specified by the cross-reference path; it's omitted if you're supposed to look in the same module. This will be made clearer in the future when everything gets refactored as @milseman suggested.